common/src/reserved.rs

```diff
@@ -338,4 +408,231 @@ mod tests {
                 .collect::<HashSet<_>>()
         );
     }
+
+    #[test]
+    fn test_apply_delegate_on_governance_success() {
+        // given
+        setup_test();
+        let (mut state, keys) = generate_standard_genesis(3);
```

Comment:
```suggestion
        let (mut reserved_state, keys) = generate_standard_genesis(4);
```
`reserved_state` is more precise and rest of the codes use this name so let's fix it this way.
Also, Simperby protocol requires at 4 at minimum so even if it's a test, it might be misleading so let's fix to 4.

Reply:
okay. I would change it.